app/javascript/gobierto_dashboards/webapp/lib/utils.js

```diff
@@ -0,0 +1,10 @@
+export function formatCurrency(amount) {
```

Comment:
No need to define a new function here, there's a filter for currencies in `app/javascript/lib/shared/modules/vue-filters.js`. Filters can be used in the HTML of a component like these {{ amount | money }}